CIP-Draft-Initial-Params/Initial.md

```diff
@@ -0,0 +1,172 @@
+## Preamble
+
+---
+* CIP: ?
+* Title: Cardano Protocol Parameters
+* Authors: Kevin Hammond (<kevin.hammond@iohk.io>)
+* Discussions-To: <kevin.hammond@iohk.io>
+* Comments-Summary: <summary tone>
+* Comments-URI: <links to wiki page for comments>
+* Status: Draft
+* Type: Standards Track
+* Created: 2020-10-14
+* License: <abbreviation for approved license(s)>
+* License-Code: <abbreviation for code under different approved license(s)>
+* Post-History: <dates of postings to Cardano Dev Forum, or link to thread>
+* Requires: <CIP number(s)>
+* Replaces: <CIP number>
+* Superseded-By: <CIP number>
+---
+
+## Simple Summary/Abstract
+
+This CIP describes the initial protocol parameter settings for the Shelley era of the Cardano blockchain.
+
+## Motivation
+
+We need to provide a concise description of the initial protocol parameter choices, that can be used as the base for future proposed protocol changes.
+
+## Specification
+
+The initial protocol parameters are given below (in JSON format):
```

Comment:
Maybe add a clear disclaimer that the values below may not reflect the current mainnet values (as for k for example).

Reply:
OK

Reply:
I've included the changes, so I don't think a disclaimer is needed now.  However, someone will need to maintain it!